pyproject.toml

```diff
@@ -1,8 +1,132 @@
+description="Library and CLI tool for analysing CloudFormation templates and check them for security compliance."
+readme = "README.md"
+requires-python = ">=3.8.0"
+dynamic = ["version"]
+license = { file = "LICENSE.md" }
+authors = [
+    { name = "Skyscanner Security", email = "security@skyscanner.net" }
+]
+keywords = [
+    "security",
+    "cloudformation",
+    "aws",
+    "cli"
+]
+
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Security"
+]
+
+dependencies = [
+    "boto3>=1.4.7,<2",
+    "cfn_flip>=1.2.0",
+    "click>=8.0.0",
+    "pluggy~=0.13.1",
+    "pycfmodel>=0.22.0",
+    "pydash>=4.7.6",
+    "PyYAML>=4.2b1"
+]
+
+[project.urls]
+documentation = "https://cfripper.readthedocs.io/"
+repository = "https://github.com/Skyscanner/cfripper"
+
+[project.scripts]
+cfripper = "cfripper.cli:cli"
+
+[project.optional-dependencies]
+dev = [
+    "moto[all]>=5",
+    "pytest-cov>=2.5.1",
+    "pytest>=3.6",
+    "ruff",
+    "uv",
+]
+docs = [
+    "mkdocs==1.3.0",
+    "mkdocs-macros-plugin==0.7.0",
+    "mkdocs-material==8.2.8",
+    "mkdocs-material-extensions==1.0.3",
+    "mkdocs-minify-plugin==0.5.0",
+]
+
+[tool.ruff]
+# Exclude a variety of commonly ignored directories.
+exclude = [
+    ".eggs",
+    ".git",
+    ".git-rewrite",
+    ".pyenv",
+    ".pytest_cache",
+    ".ruff_cache",
+    ".venv",
+    ".vscode",
+    "__pypackages__",
+    "_build",
+    "build",
+    "dist",
+    "node_modules",
+    "site",
+    "site-packages",
+    "venv",
+]
 line-length = 120
-exclude = '''
-/(
-  | \.venv
-  | venv
-)/
-'''
+indent-width = 4
+target-version = "py38"
+
+[tool.ruff.lint]
+select = ["E", "F", "W", "A", "PLC", "PLE", "PLW", "I"]
+ignore = ["A002", "E501"]
+
+# Allow fix for all enabled rules (when `--fix`) is provided.
+fixable = ["ALL"]
+unfixable = []
+
+# Allow unused variables when underscore-prefixed.
+dummy-variable-rgx = "^(_+|(_+[a-zA-Z0-9_]*[a-zA-Z0-9]+?))$"
+
+[tool.ruff.format]
+quote-style = "double"
+indent-style = "space"
+skip-magic-trailing-comma = false
+line-ending = "auto"
+docstring-code-format = false
+docstring-code-line-length = "dynamic"
+
+[tool.pytest.ini_options]
+log_cli = true
+log_level = "INFO"
+
+[tool.coverage.report]
+show_missing = true
+skip_covered = false
+
+[tool.coverage.run]
+branch = true
+source = ["cfripper"]
+
+[tool.setuptools]
+include-package-data = false
+
+[tool.setuptools.packages.find]
+# needed only because we did not adopt src layout yet
+include = ["cfripper*"]
+
+[tool.setuptools_scm]
```

Comment:
Question only: Is this needed if this section is empty?

Reply:
According to the documentation examples of the tool seems to be needed.